service/cloudconfig/cloud_config.go

```diff
@@ -0,0 +1,102 @@
+package cloudconfig
+
+import (
+	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/certificatetpr"
+	cloudconfig "github.com/giantswarm/k8scloudconfig"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	FileOwner      = "root:root"
+	FilePermission = 0700
+)
+
+// Config represents the configuration used to create a cloud config service.
+type Config struct {
+	// Dependencies.
+	Logger micrologger.Logger
+}
+
+// DefaultConfig provides a default configuration to create a new cloud config
+// service by best effort.
+func DefaultConfig() Config {
+	return Config{
+		// Dependencies.
+		Logger: nil,
+	}
+}
+
+// CloudConfig implements the cloud config service interface.
+type CloudConfig struct {
```

Comment:
This is the cloud config service to get the versioned templates. I copied that from the KVM operator to have a kind of equal interface. This just makes it work here.